deployment/terraform/agent.go

```diff
@@ -44,41 +46,84 @@ func (t *Terraform) generateLoadtestAgentConfig(output *terraformOutput) loadtes
 	}
 }
 
-func (t *Terraform) initLoadtest(extAgent *ssh.ExtAgent, ip string) error {
-	sshc, err := extAgent.NewClient(ip)
-	if err != nil {
-		return err
+func (t *Terraform) configureAndRunAgents(extAgent *ssh.ExtAgent, output *terraformOutput) error {
+	var uploadBinary bool
+	var packagePath string
+	if strings.HasPrefix(t.config.LoadTestDownloadURL, filePrefix) {
+		packagePath = strings.TrimPrefix(t.config.LoadTestDownloadURL, filePrefix)
+		info, err := os.Stat(packagePath)
+		if err != nil {
+			return err
+		}
+		if !info.Mode().IsRegular() {
+			return fmt.Errorf("load-test package path %s has to be a regular file", packagePath)
+		}
+		uploadBinary = true
 	}
 
-	mlog.Info("Populating DB", mlog.String("ip", ip))
-	cmd := "cd mattermost-load-test-ng && export PATH=$PATH:/usr/local/go/bin && go run ./cmd/ltagent init"
-	if out, err := sshc.RunCommand(cmd); err != nil {
-		return fmt.Errorf("error running ssh command: %s, out: %s, error: %w", cmd, out, err)
+	for _, val := range output.Agents.Value {
+		sshc, err := extAgent.NewClient(val.PublicIP)
+		if err != nil {
+			return err
+		}
+		mlog.Info("configuring agent", mlog.String("ip", val.PublicIP))
```

Comment:
Nit: everywhere else in this method we are starting log lines with a capital letter. Might be good to keep things uniform.